### PR TITLE
Add PC Command Tests

### PR DESCRIPTION
**Pull request by andermi**
testing:

- [x] PCWindCurrCommand
- [x] PCScaleCommand
- [x] PCRetractCommand
- [x] PCBiasCurrCommand

A few more TODO's for @hamilton8415 😜 to add a more robust fix for some NaN behavior

### CODE REVIEW

File: buoy_gazebo/src/ElectroHydraulicPTO/WindingCurrentTarget.hpp [hunk 1]
```diff
@@ -55,7 +55,7 @@ class WindingCurrentTarget
   double RetractFactor;
   double UserCommandedCurrent{0.0};
   double BiasCurrent;
-  double I{0.0};
+  mutable double I{0.0};
```
Comment (andermi) [inline buoy_gazebo/src/ElectroHydraulicPTO/WindingCurrentTarget.hpp:58]: TIL: mutable :laughing:

**Comment by andermi** [comment]
latest commit needs https://github.com/osrf/buoy_msgs/pull/18

**Comment by andermi** [comment]
also needs https://github.com/osrf/buoy_examples/pull/15

**Comment by andermi** [comment]
We will handle the TODO's in another PR

**Review by quarkytale — APPROVED** [review]
Works for me!

**Pull request merged**